MMM-Todoist.js

```diff
@@ -220,6 +223,9 @@ Module.register("MMM-Todoist", {
 					dueDateCell.innerHTML = "";
 				}
 			}
+			if (dueDateTime.getSeconds() != 59) {
```

Comment:
I'm not entirely happy with using this check to detect when a task has a due time, but it Worked For Me(tm) while testing.